frameworks/RandomForest/exec.py

```diff
@@ -16,29 +18,69 @@
 log = logging.getLogger(os.path.basename(__file__))
 
 
+def extrapolate_with_worst_case(values: List[float], n: int = 5) -> float:
```

Comment:
The mean slope would probably give a better estimate, but the maximum also sounds ok. It's not quite the worst case but possibly close enough.

Reply:
There is not much penalty for stopping a bit too early before the end of the budget, while exceeding the budget can result, in the case of memory, a failed run. For this reason I decided to use max instead of mean (which I actually started with originally). It's true that it's not the real worst case, but I feel it's a simple approximation.